### PR TITLE
feat: TEA_MAGIC=prompt

### PR DESCRIPTION
**Pull request by lino-levan**
image

### CODE REVIEW

**Comment by lino-levan** [comment]
should close #552

File: src/prefab/install.ts [hunk 1]
```diff
@@ -12,13 +12,27 @@ export default async function install(pkg: Package, logger?: Logger): Promise<In
...
 
+  if(env.TEA_MAGIC === "prompt") {
+    do {
+      const val = prompt(`┌ ⚠️  Tea requests to install ${pkg.project} (v${pkg.version})\n└ \x1B[1mAllow?\x1B[0m [y/n]`)?.toLowerCase();
+      // If val is undefined, there was no prompt given since this isn't an interactive tty
+      if(!val || val === "y") {
+        break;
+      }
+      if(val === "n") {
+        Deno.exit(1);
```
Comment (mxcl) [inline src/prefab/install.ts:30]: @ABevier are we allowed to call exit in the codebase anymore? Or we have to throw an ExitError?

Comment (ABevier) [reply src/prefab/install.ts:30]: @mxcl Ideally we'd throw an exit error.  Otherwise we can't test this path because it ends the unit test process.

Comment (lino-levan) [reply src/prefab/install.ts:30]: @ABevier could you point me to such an "Exit Error" example?

Comment (mxcl) [reply src/prefab/install.ts:30]: https://github.com/teaxyz/cli/blob/4b2c03abc6fb7c177be6ef73e05ae73eeac6aa58/src/app.exec.ts#L30

**Pull request merged**